lib/main.ts

```diff
@@ -86,9 +104,23 @@ class AchYouDataProviderEngine
     }
     return [];
   };
+
+  /**
+   * @param selector string user id or general selector
+   */
+  getUser = async (selector: string) => {
+    if (!selector) return [];
+    else if (selector === '*') return this.users;
+    else if (typeof selector === 'string') {
+      return this.users.filter((user) => {
+        return selector === user.userId;
+      });
+    }
+    return [];
+  }
 }
 
-const sneezeBoardWebViewType = "sneeze-board.sneeze-board";
+export async function activate(context: ExecutionActivationContext) {
```

Comment:
You don't need to provide the `context` parameter here if you aren't using it (because TypeScript lets you leave out parameters if you don't need to use the argument when the function is run). That being said, it won't hurt anything for this to be here either.